src/tree/CosmosDBAccountWizard/CosmosDBAccountApiStep.ts

```diff
@@ -0,0 +1,32 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AzureWizardStep, IAzureUserInput, IAzureQuickPickItem } from 'vscode-azureextensionui';
+import { ICosmosDBWizardContext } from './ICosmosDBWizardContext';
+import { Experience, DBAccountKind } from '../../constants';
+
+export class CosmosDBAccountApiStep extends AzureWizardStep<ICosmosDBWizardContext> {
+    public async prompt(wizardContext: ICosmosDBWizardContext, ui: IAzureUserInput): Promise<ICosmosDBWizardContext> {
+        const picks: IAzureQuickPickItem<string>[] = [
+            { label: Experience.MongoDB, description: '', data: DBAccountKind.MongoDB },
+            { label: Experience.DocumentDB, description: '', data: DBAccountKind.GlobalDocumentDB },
+            { label: Experience.Graph, description: '', data: DBAccountKind.GlobalDocumentDB },
```

Comment:
Likewise Graph and Table are shown differently in the portal now:

![image](https://user-images.githubusercontent.com/6913354/37675581-d4b0f5e6-2c32-11e8-85da-b75afce5ef9f.png)